pkgs/top-level/all-packages.nix

```diff
@@ -30109,7 +30109,7 @@ with pkgs;
 
   cmctl = callPackage ../applications/networking/cluster/cmctl { };
 
-  cmus = callPackage ../applications/audio/cmus {
+  cmus = callPackage ../by-name/cm/cmus/package.nix {
```

Comment:
Putting things into `by-name` and then still keeping this in `all-packages.nix` will import the derivation twice, wasting compute cycles and memory.

Reply:
[pkgs/by-name/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md#changing-implicit-attribute-defaults) allows use of this pattern when attribute arguments are changed. I think this is required for Darwin libraries.

Reply:
Indeed, that section's guidance is counterproductive as it will force that `by-name` file to be loaded twice by Nix, wasting resources. Nixpkgs takes ridiculous amounts of RAM to evaluate already, no need to make it worse, IMHO.

Reply:
Okay I removed the migration commit. If I left it in the by-name directory I'd have to name it something unconventional like default.nix or cmus.nix.